Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,262 @@
+                if (!this._color) {
+                    this._color = new Color();
+                }
+                return this._content.batchTable.getColor(this._batchId, this._color);
+            },
+            set : function(value) {
+                this._content.batchTable.setColor(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets the feature content.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Cesium3DTileContent}
+         *
+         * @readonly
+         */
+        content : {
+            get : function() {
+                return this._content;
+            }
+        }
+    });
+
+    /**
+     * Returns whether the feature contains this property. This includes properties from this feature's
+     * class and inherited classes, in addition to the standard batch table properties.
+     * <p>
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color} are not equivalent to
+     * <code>'show'</code> and <code>'color'</code> properties; the former are runtime-specific properties
+     * that are not part of the feature's properties in the stored 3D Tileset.
+     * </p>
+     *
+     * @param {String} name The case-sensitive name of the property.
+     * @returns {Boolean} Whether the feature contains this property.
+     */
+    Cesium3DTileFeature.prototype.hasProperty = function(name) {
+        return this._content.batchTable.hasProperty(this._batchId, name);
+    };
+
+    /**
+     * Returns an array of property names for the feature. This includes properties from this feature's
+     * class and inherited classes, in addition to the standard batch table properties.
+     * <p>
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color} are not equivalent to
+     * <code>'show'</code> and <code>'color'</code> properties; the former are runtime-specific properties
+     * that are not part of the feature's properties in the stored 3D Tileset.
+     * </p>
+     *
+     * @returns {String[]} The names of the feature's properties.
+     */
+    Cesium3DTileFeature.prototype.getPropertyNames = function() {
+        return this._content.batchTable.getPropertyNames(this._batchId);
+    };
+
+    /**
+     * Returns the value of the feature's property with the given name.
+     * <p>
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color} are not equivalent to
+     * <code>'show'</code> and <code>'color'</code> properties; the former are runtime-specific properties
+     * that are not part of the feature's properties in the stored 3D Tileset.
+     * </p>
+     *
+     * @param {String} name The case-sensitive name of the property.
+     * @returns {*} The value of the property or <code>undefined</code> if the property does not exist.
+     *
+     * @example
+     * // Display all the properties for a feature in the console log.
+     * var properties = tileset.properties;
```

Comment:
Would it be better for this example to use `getPropertyNames`?  I probably wrote this example before that function existed.